File: Watch Store App/WatchStoreApp-Frontend/watchstoreapp/src/Model/customer.ts
export class customer {
  constructor(
    public emailId: string = '',
    public password: string = '',
    public firstName: string = '',
    public lastName: string = '',
    public role: string = '',
    public inCartNo: number = 0,
    public inCart: string = ''
  ) {}

  toString() {
    return (
      'Email Id is:- ' +
      this.emailId +
      'Paswword is:- ' +
      this.password +
      'First Name is:- ' +
      this.firstName +
      'Last Name is:- ' +
      this.lastName +
      'Role is:- ' +
      this.role +
      'in cart no is:- ' +
      this.inCartNo +
      'in cart is:- ' +
      this.inCart
    );
  }
}

File: Watch Store App/WatchStoreApp-Backend/watchstoreapp/src/main/java/com/mywatch/entity/Customer.java
package com.mywatch.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name="tbl_customer")
public class Customer {
	
	@Id
	private String emailId;
	@Column
	private String password;
	@Column
	private String firstName;
	@Column
	private String lastName;
	@Column
	private String role;
	@Column
	private int inCartNo;
	@Column
	private String inCart;
	public String getEmailId() {
		return emailId;
	}
	public void setEmailId(String emailId) {
		this.emailId = emailId;
	}
	public String getPassword() {
		return password;
	}
	public void setPassword(String password) {
		this.password = password;
	}
	public String getFirstName() {
		return firstName;
	}
	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}
	public String getLastName() {
		return lastName;
	}
	public void setLastName(String lastName) {
		this.lastName = lastName;
	}
	public String getRole() {
		return role;
	}
	public void setRole(String role) {
		this.role = role;
	}
	public int getInCartNo() {
		return inCartNo;
	}
	public void setInCartNo(int inCartNo) {
		this.inCartNo = inCartNo;
	}
	public String getInCart() {
		return inCart;
	}
	public void setInCart(String inCart) {
		this.inCart = inCart;
	}
	@Override
	public String toString() {
		return "Customer [emailId=" + emailId + ", password=" + password + ", firstName=" + firstName + ", lastName="
				+ lastName + ", role=" + role + ", inCartNo=" + inCartNo + ", inCart=" + inCart + "]";
	}
	
	
	
}
